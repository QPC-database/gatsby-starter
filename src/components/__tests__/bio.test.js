import React from "react";
import renderer from "react-test-renderer";
import { render } from "react-testing-library"
import Bio from "../bio";

describe("Bio", () =>
  it("renders correctly -- according to react-test-renderer", () => {
    const tree = renderer.create(<Bio />).toJSON();
    expect(tree).toMatchSnapshot();
  }),
  it("renders correctly -- according to react-testing-library", () => {
    const { getByTestId } = render(<Bio />);
    expect(getByTestId("my-bio")).toHaveTextContent("My Bio!");
  })
)
