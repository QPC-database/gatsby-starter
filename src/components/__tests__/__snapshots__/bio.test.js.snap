// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio renders correctly -- according to react-test-renderer 1`] = `
<p
  data-testid="my-bio"
>
  It's nice to meet you!
</p>
`;
